### PR TITLE
AWSO 2.9.0 Docs update

### PR DESCRIPTION
**Pull request by akhil-sumologic**
## Purpose of this pull request

Added new dashboards for MSSQL in Amazon RDS section for AWSO, and incorporated the AWSO 2.9.0 related changes.

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

https://sumologic.atlassian.net/browse/SUMO-240446

### CODE REVIEW

File: docs/integrations/amazon-aws/rds.md [hunk 1]
```diff
@@ -660,7 +660,7 @@ The **Amazon RDS - PostgreSQL Logs - Slow Query Overview** dashboard provides an
 
 Use this dashboard to:
 * Obtain count of slow queries and unique slow queries.
-* Identify number of Slow queries by user,host and command type along with slow queries over time by user and database.
+* Identify number of Slow queries by user, host and command type along with slow queries over time by user and database.
```
Comment (JV0812) [inline docs/integrations/amazon-aws/rds.md:663]:
```suggestion
* Identify number of Slow queries by user, host, and command type along with slow queries over time by user and database.
```

File: docs/observability/aws/deploy-use-aws-observability/configure-alerts.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ Sumo Logic provides the following out-of-the-box alerts:
 | Amazon RDS PostgreSQL - High Errors	| This alert fires when we detect high number (>10) of error/fatal logs in Postgres logs over a 5 minutes time-period. | &#62;10 | &#60;&#61;10 |
 | Amazon RDS PostgreSQL - Statement Timeout | This alert fires when we detect Postgres logs show statement timeouts. | &#62;0 | &#60;&#61;0 |
 | Amazon RDS PostgreSQL - Excessive Slow Query Detected | This alert fires when we detect the average time to execute a query is more than 5 seconds over a 10 minutes. | &#62;&#61;1 | &#60;1 |
+| Amazon RDS MSSQL - Database observing authentication failures from multiple client IPs | This alert fires when we detect more than or equal to 10 client IPs attempting authentication failures on the database over a 15-minute period. | &#62;&#61;1 | &#60;1 |
```
Comment (JV0812) [inline docs/observability/aws/deploy-use-aws-observability/configure-alerts.md:41]:
```suggestion
| Amazon RDS MSSQL - Database observing authentication failures from multiple client IPs | This alert fires when we detect more than or equal to 10 client IPs attempting authentication failures on the database over a 15 minute time-period. | &#62;&#61;1 | &#60;1 |
```

File: docs/observability/aws/integrations/amazon-rds.md [hunk 3]
```diff
@@ -11,16 +11,16 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
 * [Amazon RDS operations using AWS CloudTrail](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/logging-using-cloudtrail.html).
 * [Publishing RDS CloudWatch Logs, RDS Database logs for Aurora MySQL, RDS MySQL, MariaDB](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.MySQLDB.PublishtoCloudWatchLogs.html).
-* [Publishing RDS CloudWatch logs, RDS Database logs for Aurora PostgreSQL, and RDS PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.PostgreSQL.html#USER_LogAccess.Concepts.PostgreSQL.PublishtoCloudWatchLogs).
-
+* [Publishing RDS CloudWatch logs, RDS Database logs for Aurora PostgreSQL, RDS PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.PostgreSQL.html#USER_LogAccess.Concepts.PostgreSQL.PublishtoCloudWatchLogs)
```
Comment (JV0812) [inline docs/observability/aws/integrations/amazon-rds.md:22]:
```suggestion
* [Publishing RDS CloudWatch logs, RDS Database logs for Aurora PostgreSQL, and RDS PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_LogAccess.Concepts.PostgreSQL.html#USER_LogAccess.Concepts.PostgreSQL.PublishtoCloudWatchLogs)
```

File: docs/observability/aws/integrations/amazon-rds.md [hunk 4]
```diff
@@ -461,20 +479,20 @@ Use this dashboard to:
 
 The **Amazon RDS - PostgreSQL Logs - Slow Query Overview** dashboard provides an overview of the slow query logs. AWS RDS will report slow logs with statement taking more than threshold value given through `log_min_duration_statement`. This dashboard can be filtered with different values for query execution time through `slow_query_latency_ms`.
 
-Use this dashboard to: 
+Use this dashboard to:
 * Obtain count of slow queries and unique slow queries.
-* Identify number of slow queries by user, host, and command type along with slow queries over time by user and database.
+* Identify number of Slow queries by user, host and command type along with slow queries over time by user and database.
```
Comment (JV0812) [inline docs/observability/aws/integrations/amazon-rds.md:484]:
```suggestion
* Identify number of slow queries by user, host, and command type along with slow queries over time by user and database.
```

Comment (JV0812) [reply docs/observability/aws/integrations/amazon-rds.md:484]: There is no UI support here to capitalize "S" here. Slow Query Overview is the dashboard title and that is covered in the first paragraph.

File: docs/observability/aws/integrations/amazon-rds.md [hunk 5]
```diff
@@ -495,10 +513,32 @@ Use this dashboard to:
...
+
+Use this dashboard to:
+* Identify the authentication failures along with reason for user and client location that are used to connect.
+* Detect logon errors, including error codes, severity levels, and states.
+
+<img src={useBaseUrl('img/integrations/amazon-aws/Amazon-RDS-MSSQL-Logs-Error-Logs-Logon Analysis.png')} style={{ border: '1px solid gray' }} alt="Amazon RDS dashboard" />
+
+### 19. Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview
+
+The **Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview** dashboard provides about the hardware, authentication mode, collation, process, recent termination of SQL server, and recent creation of databases. This dashboard relies on MSSQL error logs, which need to be [enabled](#collect-amazon-rds-cloudwatch-logs) and ingested into Sumo Logic.
```
Comment (JV0812) [inline docs/observability/aws/integrations/amazon-rds.md:536]:
```suggestion
The **Amazon RDS - MSSQL Logs - Error Logs - Infrastructure Overview** dashboard provides information about the hardware, authentication mode, collation, process, recent termination of SQL server, and recent creation of databases. This dashboard relies on MSSQL error logs, which need to be [enabled](#collect-amazon-rds-cloudwatch-logs) and ingested into Sumo Logic.
```

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes

**Review by JV0812 — APPROVED** [review]
LGTM

**Review by kimsauce — APPROVED** [review]

**Comment by JV0812** [comment]
Merged the PR: https://github.com/SumoLogic/sumologic-documentation/pull/4290 in order to maintain a single PR for AWSO 2.9.0 docs changes.

File: docs/observability/aws/deploy-use-aws-observability/changelog.md [hunk 6]
```diff
@@ -9,7 +9,29 @@ This section provides details on the available versions of the AWS Observability
...
+* Integrated the latest SAM versions with AWSO v2.9.0 (CF + TF) to address CVEs and updated Lambda runtimes from Node.js v18.x to v20.x.
+* Upgraded to Sumo Logic Terraform provider v2.31.0 for CVE fixes.
+* Updated Terraform test code written in Go language from v1.18 to v1.22, addressing CVEs associated with the gRPC module.
+
+Bug Fixes:
+* Resolved an issue where updating from versions lower than v2.8.0 to v2.8.0 was not functioning correctly. Users are now recommended to update directly to v2.9.0 without needing to go through v2.8.0.
+
+## v2.8.0, 17-May-2024 - Yanked
+
+### This version has been Yanked. Please deploy/ update to AWS Observabilty v2.9.0 or latest version.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/changelog.md:34]:
```suggestion
### This version has been Yanked. Please deploy/update to AWS Observabilty v2.9.0 or latest version.
```

File: docs/observability/aws/deploy-use-aws-observability/changelog.md [hunk 7]
```diff
@@ -9,7 +9,29 @@ This section provides details on the available versions of the AWS Observability
...
-## v2.8.0, 17-May-2024
+## v2.9.0, 02-Aug-2024
+
+AWS Observability Solution (S3 Link for CloudFormation template): `https://sumologic-appdev-aws-sam-apps.s3.amazonaws.com/aws-observability-versions/v2.9.0/sumologic_observability.master.template.yaml`
+
+AWS Observability Solution (Terraform-based) repo: [sumologic-solution-templates](https://github.com/SumoLogic/sumologic-solution-templates).
+
+New Feature:
+* Amazon RDS app - Added support to analyze and monitor RDS MS SQL CloudWatch logs.
+* Amazon RDS app - New monitors added for RDS MS SQL CloudWatch logs. Solution now supports 70 out of box monitors.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/changelog.md:20]:
```suggestion
* Amazon RDS app - New monitors added for RDS MS SQL CloudWatch logs. Solution now supports 70 out-of-box monitors.
```

File: docs/observability/aws/deploy-use-aws-observability/changelog.md [hunk 8]
```diff
@@ -9,7 +9,29 @@ This section provides details on the available versions of the AWS Observability
...
+* By default, the solution now sends CloudFormation deployment telemetry to Sumo Logic. Users have the option to opt out of this telemetry.
+
+Updates:
+* Reduced the installation time for the AWS Observability solution using the CloudFormation template.
+* Integrated the latest SAM versions with AWSO v2.9.0 (CF + TF) to address CVEs and updated Lambda runtimes from Node.js v18.x to v20.x.
+* Upgraded to Sumo Logic Terraform provider v2.31.0 for CVE fixes.
+* Updated Terraform test code written in Go language from v1.18 to v1.22, addressing CVEs associated with the gRPC module.
+
+Bug Fixes:
+* Resolved an issue where updating from versions lower than v2.8.0 to v2.8.0 was not functioning correctly. Users are now recommended to update directly to v2.9.0 without needing to go through v2.8.0.
```
Comment (kimsauce) [inline docs/observability/aws/deploy-use-aws-observability/changelog.md:30]:
```suggestion
* Resolved an issue where updating from versions lower than v2.8.0 to v2.8.0 was not functioning correctly. Users are now recommended to update directly to v2.9.0, bypassing v2.8.0.
```

**Pull request merged**